src/ghdcbot/adapters/storage/sqlite.py

```
@@ -0,0 +1,655 @@
+        if now - verified_at < cooldown:
+            remaining = (verified_at + cooldown) - now
+            total_seconds = int(remaining.total_seconds())
+            hours, rem = divmod(total_seconds, 3600)
+            minutes, _ = divmod(rem, 60)
+            if hours > 0:
+                remaining_str = f"{hours}h {minutes}m"
+            else:
+                remaining_str = f"{minutes}m"
+            raise ValueError(
+                f"Identity was verified recently. You can unlink after {remaining_str}."
+            )
```

Ruff (0.14.14)

[warning] Avoid specifying long messages outside the exception class

(TRY003)